test/expression_utils_test.py

```diff
@@ -128,3 +133,17 @@ def test_get_expression_levels_zero_sum_fpkm(self, _get_feature_ids):
         for tpm in tpm_dict.values():
             sum_tpm += tpm
         self.assertEquals(0.0, sum_tpm)
+
+    @patch.object(ExpressionUtils, "_get_feature_ids",
+                  side_effect=mock_get_feature_ids)
+    def test_get_transcript_expression_levels(self, _get_feature_ids):
+        exp_utils = ExpressionUtils(self.__class__.cfg)  # no logger specified
+
+        fpkm_dict, tpm_dict = exp_utils.get_expression_levels(
+            filepath='data/expression_utils/t_data.ctab',
+            genome_ref='', id_col=5)
+
+        print(fpkm_dict, tpm_dict)
+
+        self.assertEquals(9, len(fpkm_dict))
```

Comment:
Also compare the values within the fpkm, tpm dictionaries with expected values to check accuracies of results?